js/05/main.js

```diff
@@ -0,0 +1,47 @@
+// First remove all the punctuations and change the string to array and count the number of words in the array
```

Comment:
We also need to remove punctuations here. Could you please check once again the condition of task.